### PR TITLE
Mobile nav updated

### PR DESCRIPTION
**Pull request by akashvenus**
## Changes proposed

- Mobile view nav bar spacing updated

## Check List (Check all the applicable boxes) 

- [x ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x ] This PR does not contain plagiarized content.
- [x ] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot (89)

### CODE REVIEW

File: package-lock.json
Comment (Cahllagerfeld) [inline package-lock.json:1]: Could you please revert these changes

File: src/lib/components/hamburger-menu/menu.svelte [hunk 1]
```diff
@@ -48,7 +48,7 @@
         {/if}
         <ul>
           {#each navItems as item}
-            <li>
+            <li class="mt-9 px-3">
```
Comment (Cahllagerfeld) [inline src/lib/components/hamburger-menu/menu.svelte:51]: I feel this margin plus a padding is a bit too large what do you think?

File: package-lock.json
Comment (akashvenus) [reply package-lock.json:1]: Yes I can. Should I reduce the padding? I included it incase the person touches an cm to the side of the content.

File: src/lib/components/hamburger-menu/menu.svelte (hunk 1 above)
Comment (akashvenus) [reply src/lib/components/hamburger-menu/menu.svelte:51]: Maybe I can reduce the padding. Just wanted to ask you regarding this, since there only two options in the menu

**Comment by akashvenus** [comment]
Hi there @Cahllagerfeld I've reverted the package-lock.json changes. Idk why it removed 5000 + lines. Also, I've fixed the margin spacing. 

image

**Comment by akashvenus** [comment]
@Cahllagerfeld could you please check the updates

File: src/lib/components/hamburger-menu/menu.svelte [hunk 2]
```diff
@@ -48,7 +48,7 @@
         {/if}
         <ul>
           {#each navItems as item}
-            <li>
+            <li class="mt-6 px-3">
```
Comment (Cahllagerfeld) [inline src/lib/components/hamburger-menu/menu.svelte:51]: Instead of having `mt-6` applied here, you can also try using `space-y-6` or `space-y-4` on the `<ul>` element. This is a bit cleaner in my opinion

File: src/lib/components/hamburger-menu/menu.svelte [hunk 3]
```diff
@@ -61,7 +61,7 @@
         {#if !username}
           <a
             data-test-id="login-btn"
-            class="default-transition rounded-xl bg-skin-text px-4 py-2 text-skin-off-background hover:bg-primary-100 dark:bg-skin-text-highlight dark:text-skin-background dark:hover:bg-primary-100"
+            class="default-transition mt-5 rounded-xl bg-skin-text px-4 py-2 text-skin-off-background hover:bg-primary-100 dark:bg-skin-text-highlight dark:text-skin-background dark:hover:bg-primary-100"
```
Comment (Cahllagerfeld) [inline src/lib/components/hamburger-menu/menu.svelte:64]: I think we should be consistent with margins and use the same number here as above

File: package-lock.json
Comment (Cahllagerfeld) [inline package-lock.json:1]: The package-lock shouldn't be touched in that PR, could you please revert it 🙏

**Review by Cahllagerfeld — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (akashvenus) [reply package-lock.json:1]: I did revert it. Didn't it change?

File: src/lib/components/hamburger-menu/menu.svelte (hunk 2 above)
Comment (akashvenus) [reply src/lib/components/hamburger-menu/menu.svelte:51]: Seems right. Will make the changes

File: src/lib/components/hamburger-menu/menu.svelte (hunk 3 above)
Comment (akashvenus) [reply src/lib/components/hamburger-menu/menu.svelte:64]: Got it. Will change the following

**Comment by akashvenus** [comment]
@Cahllagerfeld Could you please check the changes

**Review by Cahllagerfeld — APPROVED** [review]
Lets merge it!
